packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -33,6 +35,21 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
   name: 'JiraServerIntegration',
   description: 'Jira Server integration data for a given team member',
   fields: () => ({
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'Composite key in jiraServer:providerId format',
+      resolve: async ({teamId, userId}: {teamId: string; userId: string}, _args, {dataLoader}) => {
+        const auth = await dataLoader
+          .get('teamMemberIntegrationAuths')
+          .load({service: 'jiraServer', teamId, userId})
+
+        if (!auth) {
+          return ''
```

Comment:
Why did you add the `id` field?

Reply:
@Dschoordsch it is needed for relay to correctly update the store. We request `JiraServerIntegration.searchQueries` in persist search query mutation. So relay updates `searchQueries` automatically